armonik-client/src/main/java/fr/aneo/armonik/client/internal/grpc/observers/UploadBlobDataObserver.java

```diff
@@ -189,4 +215,15 @@ private void sendIds() {
     request.onNext(toUploadResultDataIdentifierRequest(sessionId, blobId));
     headerSent = true;
   }
+
+  private void closeStream() {
```

Comment:
Seems like a bug and another PR to me

Reply:
This method is about to close the stream(InputStream) from BlobData. In the context of this Observer there is only one stream. But I can rename the field `inputStream` to `dataInputStream` and this method to `closeDataInputStream`

Reply:
I don't think the renaming is needed